### ISSUE TITLE
A question about the loss of RCNN

### ISSUE THREAD

**Issue by DehuaTang** [yhenon/keras-frcnn#234]
loss_class = model_classifier.train_on_batch([X, X2[:, sel_samples, :]], [Y1[:, sel_samples, :], Y2[:, sel_samples, :]])

losses[iter_num, 0] = loss_rpn[1]
losses[iter_num, 1] = loss_rpn[2]

losses[iter_num, 2] = loss_class[1]
losses[iter_num, 3] = loss_class[2]
losses[iter_num, 4] = loss_class[3]

Thank you for sharing your work !
In the train.py ,  the "loss_class " is a tensor with the shape of [1 x 4].
Can you tell me what is "loss_class[3]" ?   Where did you define it ?
Thank you !
